packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,55 @@
+Sometimes there may be a need to render a list from something other than an
```

Comment:
I think there's a question of ordering here... is .map(), imperative, map(), repeat(), the best order? map() is probably 99% of the time more readable than imperative, so maybe it should come first?

Reply:
That makes sense. I felt that starting with `map()` then reverting to imperative for the set would be silly, but perhaps the imperative step could be combined with the heterogeneous list idea below and to follow `map()`? What do you think about the placing of the events step? Before or after `repeat()`? I feel it could go either way, but leaning toward after.